docs/generated/packages/jest/executors/jest.json

```diff
@@ -166,15 +170,17 @@
       "updateSnapshot": {
```

Comment:
Same here. Obviously you're the expert so I'll trust your judgement. But I would think that folks would rather use `-u` when they want to update snapshots and not set it every time...